src/Altinn.Platform/Altinn.Platform.Register/Register/Controllers/PersonsController.cs

```diff
@@ -25,18 +26,21 @@ public PersonsController(IPersons personsWrapper)
         /// <summary>
         /// Gets the person for a given ssn
         /// </summary>
-        /// <param name="ssn">The ssn</param>
         /// <returns>The information about a given person</returns>
-        [HttpPost]
-        public async Task<ActionResult> Get([FromBody]string ssn)
+        [HttpGet]
+        public async Task<ActionResult> Get()
```

Comment:
Why can you not use FromBody in the Get?

Reply:
Hmm, didn't try that... Probably works. I'll test.